package.json

```diff
@@ -49,6 +49,7 @@
   "dependencies": {
     "electron-compile": "^6.4.4",
     "electron-squirrel-startup": "^1.0.0",
+    "mkdirp": "^0.5.1",
```

Comment:
Is this dependency actually needed?

Reply:
Needed it for smth :/ Added this to git ignore